TTS/utils/synthesizer.py

```diff
@@ -122,6 +122,13 @@ def tts(self, text, speaker_idx=None):
         speaker_embedding = self.init_speaker(speaker_idx)
         use_gl = self.vocoder_model is None
 
+
+        # check if compute gst style
```

Comment:
maybe we can make GST style wav as an additional command argument to make things easy-to-use

Reply:
I let it for now, and am planning to make a new PR for resolving this GST + speaker embedding tuning when using the Synthetizer class (used in server.py script)